grid.css

```diff
@@ -0,0 +1,20 @@
+.grid-container {
+  display: grid;
+  grid-gap: 1.2rem;
+  min-height: 6.5rem;
+  grid-template-columns: 1fr;
+}
+@media screen and (min-width: 768px) {
+  .grid-2 {
+    grid-template-columns: 1fr 1fr;
+  }
+  .grid-3 {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+.grid-content {
+  display: flex;
+  position: relative;
```

Comment:
does this position relative have any particular reason to be here? i can't tell, so do look into it.

Reply:
Removed